docs/modules/ima-ads/README.md

```diff
@@ -2,11 +2,25 @@
 
 Module to display VAST compatible advertisement with [Google IMA library](https://developers.google.com/interactive-media-ads/docs/sdks/html5/).
 
-To use this module you need to add this script to your `head` tag on the `index.html`:
+To use this module you need to add this script to your `head` tag and initialize the googletag property before the declaration on the `index.html`:
 
 ```html
+<script>
+    var googletag = googletag || {};
+    googletag.cmd = googletag.cmd || [];
+    (function() {
+    var gads = document.createElement('script');
+    gads.async = true; gads.type = 'text/javascript';
+    gads.src = 'https://www.googletagservices.com/tag/js/gpt.js';
+    var node = document.getElementsByTagName('script')[0];
```

Comment:
Consider using `let` rather than `var`. 

**Reference**:
- https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable